src/frontend/web_application/src/scenes/Timeline/components/MessageItem/presenter.jsx

```diff
@@ -147,7 +153,7 @@ class MessageItem extends Component {
           }
         )}
       >
-        <div className="s-message-item__col-avatar">
+        <div className="s-message-item__column--avatar">
```

Comment:
hmm. this can be sort of modifier but I think not:

Ok all of these are columns but in this case the name of the modifier should be "shrink" or "large" or ...
But finally this doesn't give any advantages:

```scss
&__column { 
  // no generic rule
  &--avatar { // is the name of the column (not a modifier name)
    // specific rule 
  }
}
```

I prefer what was before and imo more in the concept of the Element in BEM:

```scss
&__col-<the name of the column> {
 // the directives of the column
}
```

Or... if you prefer to go to refactor (but overkill for now):

```
<Column className="s-message-item__avatar">....
```

---

Eventually if it is required for some reasons, please leave a comment in code explaining why we needed to break the "rules" (because that's ok ;-) )